Gemfile

```diff
@@ -2,8 +2,7 @@ source 'https://rubygems.org'
 
 gem 'rails', '4.1.6'
 
-gem 'krikri', github: 'dpla/krikri', branch: 'develop'
-gem 'marmottawrapper', '~>0.0.5'
+gem 'krikri', github: 'dpla/krikri', branch: 'fix-jettywrapper-in-host'
```

Comment:
This can be changed back to the `develop` branch, since dpla/KriKri#47 was merged.